docs/pages/push-notifications/faq.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Push Notifications Troubleshooting & FAQ
+sidebar_title: Troubleshooting and FAQ
+---
+
+This is a collection of FAQs and common issues when setting up push notifications with Expo. This document covers the `expo-notifications` client-side library, as well as Expo's push notification service.
+
+### How much does Expo's push notification service cost?
+
+There is no cost associated with sending notifications through Expo's push notification service.
+
+### How many notifications can I send through Expo?
+
+We don't impose any limit on the number of push notifications you can send. For best results, we do recommend you add throttling (handled automatically in the [`expo-server-sdk-node`](https://github.com/expo/expo-server-sdk-node)) and retry logic to your server.
+
+Total notifications is much less important than your maximum notification throughput- how many notifications you send per second at peak. If this number is more than a couple hundred, reach out to us because we'd love to hear about what you're working on!
```

Comment:
```suggestion
Total notifications is much less important than your maximum notification throughput—how many notifications you send per second at peak. If this number is more than a couple hundred, reach out to us because we'd love to hear about what you're working on!
```